JS/maps.js

```diff
@@ -0,0 +1,34 @@
+var map = new Map();
+map.set("r1", "user").set("r2", "admin");
+/*console.log(map);*/
```

Comment:
Remove the commented code if not required.

Reply:
tried all the different methods for map

Reply:
only for reference, instead of creating no. of files I executed them one by one